docs/terminology.md

```diff
@@ -273,7 +273,7 @@ header, array of account addresses, recent [blockhash](#blockhash), and an array
 of [instructions](#instruction).
 
 Learn more about the
-[message formatting inside of transactions](/docs/core/transactions.md#message-format)
+[message formatting inside of transactions](/docs/core/transactions#transaction)
```

Comment:
linking to the actual message header makes more sense

Reply:
updated